### PR TITLE
Docker linking replaced by networking, and redis image now uses alpine

### PR DESCRIPTION
**Pull request by lpalmes**
## Changes
- scripts checks if docker is already running for linux and docker native env
- ENV have been removed, @davidpdrsn but i just realized you might use them for production, if that's so let me know and i turn back to using env
- While trying to make it work i used a random secret_key_base in config/secrets.yml, that should be ignored
- redis now uses alpine version and it benefits from a smaller image size

```
redis                 2.6                 a081f7d44c38        5 months ago        150.2 MB
redis                 alpine              8a8bdae38335        8 days ago          29.07 MB
```
- i changed the logo path from `logo2.svg` to `logo.svg`

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 default: &default
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 8370c4c3531fe72f6988d1a90fc8aa0be2f38492340ac7ec78cf7875797e19bd788e28a1e28ffebb261ba958c387298b082b708c45e46922c1268b16ea5eff15
```
Comment (davidpdrsn) [inline config/secrets.yml:2]: For security reasons it is not recommended to commit the secret key base. You can find more info [here](http://stackoverflow.com/questions/25426940/what-is-the-use-of-secret-key-base-in-rails-4)

File: docker-compose.yml [hunk 2]
```diff
@@ -1,48 +1,58 @@
...
+    build: .
+    command: echo "Hello, world!"
+    volumes:
+      - .:/app
+    stdin_open: true
+    environment:
+      BUNDLE_PATH: "/box"
+      SKYLIGHT_AUTHENTICATION: "test"
+      INSTAGRAM_CLIENT_ID: "test"
+      INSTAGRAM_CLIENT_SECRET: "test"
```
Comment (davidpdrsn) [inline docker-compose.yml:15]: These environment variables will be set by the ".env" file which is created as part of the bootstrap script. You might have to rebase master to get the change.

File: script/build [hunk 3]
```diff
@@ -7,12 +7,14 @@ rm -rf log/test.log
 touch log/development.log
 touch log/test.log
 
-if [ $(uname) = "Darwin" ]; then
-  echo "---- Making sure docker-machine is running"
-  docker-machine start default || true
+if ! command -v docker > /dev/null; then
```
Comment (davidpdrsn) [inline script/build:10]: Why is this extra conditional needed? We should probably exit if docker is not installed.

**Comment by davidpdrsn** [comment]
I had a questions and comments, otherwise it looked good 👍

File: config/secrets.yml (hunk 1 above)
Comment (lpalmes) [inline config/secrets.yml:2]: I just put a random string hash cause i was struggling with getting the app up and runnning with docker, this should be ignored

File: docker-compose.yml (hunk 2 above)
Comment (lpalmes) [inline docker-compose.yml:15]: I also did use this random env for getting up and running

File: script/build (hunk 3 above)
Comment (lpalmes) [inline script/build:10]: This is for linux environments which already have docker without a docker-machine, and also if docker already available just run it

**Comment by lpalmes** [comment]
I will close this one and make a cleaner pr

**Pull request closed without merging**